### DISCUSSION TITLE
Phoscon App not finding conbee II

### DISCUSSION THREAD

**Discussion by jddayley** [tteck/Proxmox#288] [General]
The phoscon app is not able to detect the conbee II.   I can see it on the proxmox host when I run lsusb.   I also see it in the lxc container when I run ls -l /dev/serial/by-id/  
/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2495756-if00 -> ../../ttyACM0

I also tried to upgrade the firmware.

root@deconz:/usr/share/deCONZ/firmware# GCFFlasher_internal -d /dev/ttyACM0 -f deCONZ_ConBeeII_0x26720700.bin.GCF -x 3
GCFFlasher V3_17 (c) dresden elektronik ingenieurtechnik gmbh
11:16:09:329 using firmware file: deCONZ_ConBeeII_0x26720700.bin.GCF 
11:16:09:388 ls dev: /dev/ttyACM0 (0x0000/0x0000) sn: 
11:16:09:388 ls dev: /dev/ttyS0 (0x0000/0x0000) sn: 
11:16:09:388 ls dev: /dev/ttyS1 (0x0000/0x0000) sn: 
11:16:09:388 device /dev/ttyACM0 not found
root@deconz:/usr/share/deCONZ/firmware#

**Comment by tteck** [comment]
Is the LXC Privileged?

**Comment by tteck** [comment]
> I also see it in the lxc container when I run ls -l /dev/serial/by-id/
> /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2495756-if00 -> ../../ttyACM0

I'd say yes... lol

**Comment by tteck** [comment]
Beings that I don't own a Conbee II stick, hopefully someone with experience will chime in and help

**Comment by jddayley** [comment]
Is there a command that I could run to determine that information?  Otherwise, I'm going to reinstall the script

**Comment by tteck** [comment]
Look under the LXC's options tab

![Screenshot 2022-06-07 7 34 37 AM](https://user-images.githubusercontent.com/21342670/172370059-54085235-f950-4f67-b552-e399b151371c.png)

**Comment by jddayley** [comment]
<img width="391" alt="Screen Shot 2022-06-07 at 7 38 13 AM" src="https://user-images.githubusercontent.com/29357870/172370553-0f0e3533-132e-4f62-9a16-e415dfec75e9.png">

**Comment by schiz0phr3ne** [comment]
Did you tried to update your Conbee II with her ID (`/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2495756-if00`) ?

In Zigbee2MQTT, we can use ID or logical name.

**Comment by nicedevil007** [comment]
Can you post your entire LXC config file here?
To get the conbee up and running wasn't an easy task inside a LXC but I got it working on a friends environment where I still have access to :)

Maybe I can help out.

**Comment by MrThiemann** [comment]
Hey 👋
I actually wanted to try your script too.... but I can't get my conbee2 to run.
In the HA integration it works perfectly?!

Do I need to make big changes to your /etc/pve/lxc/ID.conf?
everything I can find on the net shows me that I have to enter some values here.

![Bildschirmfoto 2022-10-02 um 22 54 07](https://user-images.githubusercontent.com/54147030/193476221-9d0d1c5c-6a9c-44ed-bbf9-84a68201798d.png)

if I edit the *.conf in this way, I probably get it sorted through somehow. But in decoz it still says "not connected"
![Bildschirmfoto 2022-10-02 um 22 56 40](https://user-images.githubusercontent.com/54147030/193476229-3cd654f2-95a3-45bd-8ca1-7311f0336f0e.png)

![Bildschirmfoto 2022-10-02 um 22 45 59](https://user-images.githubusercontent.com/54147030/193476259-2cb0bd52-ce4c-483e-934e-808829147035.png)
